src/journal/journald-server.c

```diff
@@ -2038,3 +2042,35 @@ int config_parse_line_max(
 
         return 0;
 }
+
+int config_parse_compress(const char* unit,
+                          const char *filename,
+                          unsigned line,
+                          const char *section,
+                          unsigned section_line,
+                          const char *lvalue,
+                          int ltype,
+                          const char *rvalue,
+                          void *data,
+                          void *userdata) {
+        JournalCompressOptions* compress = data;
+        int r;
+
```

Comment:
i think it would be great to reset this value to the default if assigned the empty string...

Reply:
I updated it to stop emitting on the empty string case, this will have the effect of using the default.